### ISSUE TITLE
Docker Saas - Sync Server Errors

### ISSUE THREAD

**Issue by hoangtuan151** [it-projects-llc/odoo-saas-tools#677]
Hi,

Saas tools is a great work. But I'm experiencing errors when sync server.
Pls help me to figure it out. Thanks in advanced!
Let me describle:

**1. Here's my script to intsall** (follow guide from [odoo-saas-tools getting-started](https://it-projects-llc.github.io/odoo-saas-tools/getting-started/)): [my_script](https://gist.github.com/hoangtuan151/b012f478d04567fa9689c711326ac1a3)

**2. Error 1: server-1 is created at the install process.**
The error occurs when I go to: SaaS => Servers => Sync server
![screenshot from 2017-11-17 07 07 04](https://user-images.githubusercontent.com/17728615/32922676-f28d7e8c-cb65-11e7-98c0-a201d86860fe.png)

Error messages:

```
Traceback (most recent call last):
  File "/mnt/odoo-source/odoo/http.py", line 640, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/mnt/odoo-source/odoo/http.py", line 677, in dispatch
    result = self._call_function(**self.params)
  File "/mnt/odoo-source/odoo/http.py", line 333, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/mnt/odoo-source/odoo/service/model.py", line 101, in wrapper
    return f(dbname, *args, **kwargs)
  File "/mnt/odoo-source/odoo/http.py", line 326, in checked_call
    result = self.endpoint(*a, **kw)
  File "/mnt/odoo-source/odoo/http.py", line 935, in __call__
    return self.method(*args, **kw)
  File "/mnt/odoo-source/odoo/http.py", line 506, in response_wrap
    response = f(*args, **kw)
  File "/mnt/odoo-source/addons/web/controllers/main.py", line 889, in call_button
    action = self._call_kw(model, method, args, {})
  File "/mnt/odoo-source/addons/web/controllers/main.py", line 877, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/mnt/odoo-source/odoo/api.py", line 689, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/mnt/odoo-source/odoo/api.py", line 680, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "<string>", line 2, in action_sync_server
  File "/mnt/odoo-source/odoo/api.py", line 369, in loop
    result = [method(rec, *args, **kwargs) for rec in self]
  File "/mnt/addons/it-projects-llc/odoo-saas-tools/saas_portal/models/saas_portal.py", line 136, in action_sync_server
    raise Warning('Reason: %s \n Message: %s' % (res.reason, res.content))
Warning: Reason: INTERNAL SERVER ERROR 
 Message: Expected singleton: saas_server.client()
```

**3. Error 2: I create new server (server-2) from Saas portal interface**
![screenshot from 2017-11-17 07 09 37](https://user-images.githubusercontent.com/17728615/32922742-4fc08360-cb66-11e7-8a99-677b579e6202.png)

Error messages when I go to: SaaS => Servers => Sync server

```
Traceback (most recent call last):
  File "/mnt/odoo-source/odoo/http.py", line 640, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/mnt/odoo-source/odoo/http.py", line 677, in dispatch
    result = self._call_function(**self.params)
  File "/mnt/odoo-source/odoo/http.py", line 333, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/mnt/odoo-source/odoo/service/model.py", line 101, in wrapper
    return f(dbname, *args, **kwargs)
  File "/mnt/odoo-source/odoo/http.py", line 326, in checked_call
    result = self.endpoint(*a, **kw)
  File "/mnt/odoo-source/odoo/http.py", line 935, in __call__
    return self.method(*args, **kw)
  File "/mnt/odoo-source/odoo/http.py", line 506, in response_wrap
    response = f(*args, **kw)
  File "/mnt/odoo-source/addons/web/controllers/main.py", line 889, in call_button
    action = self._call_kw(model, method, args, {})
  File "/mnt/odoo-source/addons/web/controllers/main.py", line 877, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/mnt/odoo-source/odoo/api.py", line 689, in call_kw
    return call_kw_multi(method, model, args, kwargs)
  File "/mnt/odoo-source/odoo/api.py", line 680, in call_kw_multi
    result = method(recs, *args, **kwargs)
  File "<string>", line 2, in action_sync_server
  File "/mnt/odoo-source/odoo/api.py", line 369, in loop
    result = [method(rec, *args, **kwargs) for rec in self]
  File "/mnt/addons/it-projects-llc/odoo-saas-tools/saas_portal/models/saas_portal.py", line 136, in action_sync_server
    raise Warning('Reason: %s \n Message: %s' % (res.reason, res.content))
Warning: Reason: NOT FOUND 
 Message: <!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
<title>404 Not Found</title>
<h1>Not Found</h1>
<p>The requested URL was not found on the server.  If you entered the URL manually please check your spelling and try again.</p>

```

I've search and read these help about not found errors, but I could not figure it out:
https://www.odoo.com/forum/help-1/question/domain-based-db-filter-6583
https://www.odoo.com/documentation/10.0/setup/deploy.html
https://github.com/it-projects-llc/odoo-saas-tools/issues/439
https://it-projects-llc.github.io/odoo-saas-tools/blog/reason-not-found

**Comment by leao-c** [comment]
Change "Request port" from 80 to 8069 or any other port your Odoo SaaS_Server is running on.

**Comment by hoangtuan151** [comment]
Hi Leao-c, 
Change Request Port to 8069 cause "open server"  redirect browser to port 8069 (for ex http://server-1.example.com:8069). But my Nginx is listened at port 80.

**Comment by njeudy** [comment]
@hoangtuan151 hello, 2- is fixed in #746 will investigate 3- one :)

**Comment by njeudy** [comment]
@hoangtuan151 you try on V10 or V11 ?
